README.md

```diff
@@ -3,45 +3,99 @@
 <!-- ABOUT THE PROJECT -->
 ## About The Project
 
-Please put here some general information about your Intergration / App / Solution.
+Xperience by Kentico Siteimprove integration allows users to check pages for issues directly from Xperience by Kentico.
+
+![Siteimprove CMS Plugin in Pages application Preview mode](/images/dancing-goat-siteimprove-plugin-admin.png?raw=true)
```

Comment:
Once we get this merged in, we can update this URL to be an absolute URL from the [GitHub Raw User content domain](https://github.com/Kentico/xperience-by-kentico-lucene/blob/b45ca6dd6e85758d65c925120f1925088757f1a7/README.md?plain=1#L14) so that it also shows up on the NuGet package page.